### PR TITLE
Create an integration test for: one fee/fine criteria - #107

### PR DESCRIPTION
**Pull request by GoZaddy**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] Check logging
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.FileSystemResource;
+
+public class OneFeeFineMatchingCriteriaTest extends BaseBatchTest {
```
Comment (ncovercash) [inline src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java:28]: Remove `public` here

File: src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+  private static final String GET_USERS_REQUEST = "/users?query=id%3D%3D%28%28bec20636-fb68-41fd-84ea-2cf910673599%20or%202205005b-ca51-4a04-87fd-938eefa8f6de%29%29&limit=50";
+
+  private static final String GET_ITEMS_REQUEST = "/inventory/items?query=id%3D%3D%28%28100d10bf-2f06-4aa0-be15-0b95b2d9f9e4%20or%20100d10bf-2f06-4aa0-be15-0b95b2d9f9e3%29%29&limit=50";
+
+  private static final String GET_TRANSFER_ACCOUNTS_REQUEST = "/lala";
```
Comment (ncovercash) [inline src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java:34]: this supposed to be /lala?

**Review by ncovercash — CHANGES_REQUESTED** [review]

File: src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java (hunk 2 above)
Comment (GoZaddy) [reply src/test/java/org/folio/dew/bursarfeesfines/OneFeeFineMatchingCriteriaTest.java:34]: Oh no, I forgot to change that back while doing some testing lol

**Review by ncovercash — APPROVED** [review]
Looking great! Thanks!

**Pull request merged**